content/docs/command-reference/config.md

```diff
@@ -238,6 +238,23 @@ to learn more about the state file (database) that is used for optimization.
   When an entry in the database is used (e.g. during the `dvc status`), DVC
   updates the timestamp on that entry. This way, when the database needs a
   cleanup, DVC can sort entries chronologically, and remove the oldest ones.
+  
+- `state.dir` - specify a custom location for the state file. This is
+  necessary when using DVC on NFS or other mounted volumes.
+  (By default, store SQLite-based remote indexes and state's
+  `links` and `md5s` caches in the repository itself to avoid any
+  possible state corruption in 'shared cache dir' scenario.) 
```

Comment:
@meierale can you clarify what this long parenthesis means? Does it intend to document the default behavior when this is not set?